frontend/packages/kubevirt-plugin/src/statuses/vm/vm-status.ts

```diff
@@ -57,6 +62,11 @@ import { PAUSED_VM_MODAL_MESSAGE } from '../../strings/vm/messages';
 const isPaused = (vmi: VMIKind): VMStatusBundle =>
   isVMIPaused(vmi) ? { status: VMStatus.PAUSED, message: PAUSED_VM_MODAL_MESSAGE } : null;
 
+const isDeleting = (vm: VMKind, vmi: VMIKind): VMStatusBundle =>
```

Comment:
+1 much better resolution than what we had before in vm.tsx

Reply:
please move the function to be in order they are called in the status

Reply:
done +1